main.go

```diff
@@ -125,5 +133,62 @@ func validateOptions(opts *options) error {
 	if opts.PProfOptions.TimeSeconds < 1 {
 		return fmt.Errorf("seconds must be an integer greater than 0")
 	}
+
+	// extra FlameGraph options
+	if opts.OutputOpts.Title == "" {
+		return fmt.Errorf("FlameGraph title should not be empty")
```

Comment:
nit: errors should be lowercase,
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

same for all errors added